modules/Components/src/client/BasicFileValidateAndSave.coffee

```diff
@@ -202,8 +202,14 @@ class BasicFileValidateAndSaveController extends Backbone.View
 		else
 			summaryStr += "Failed due to errors "
 		@notificationController.addNotifications(@errorOwnerName, json.errorMessages)
-		@$('.bv_htmlSummary').html(json.results.htmlSummary)
 		@newExperimentCode = json.results.experimentCode
+		if json.results?.htmlSummary?
+			@$('.bv_htmlSummary').html(json.results.htmlSummary)
+			@openExperimentInQueryToolController = new OpenExperimentInQueryToolController
+				code: @newExperimentCode 
+				experimentStatus: 'approved'
```

Comment:
@brianbolt when @dalejerikson and I were working on this, we structured the `OpenExperimentInQueryToolController` to take an experimentStatus as input because the behavior in Experiment Browser depended on the status, I think just hiding the Open In Button if the status wasn't approved and the viewer is LiveDesign.

SEL doesn't pass back an experiment status, so we didn't have access to the status without doing an additional query. It seems like the formally correct thing to do here would be to check the `server.sel.experimentStatus` config which sets the default status for things loaded through SEL, but with it being a "server" config, we can't access it from the client, right?

Just curious if you had thoughts here, or whether you think that leaving it hardcoded as "approved" is fine.